### ISSUE TITLE
Package linter "bool"

### ISSUE THREAD

**Issue by Shervin-QZ** [YunoHost/package_linter#175]
### Hi while testing by yunohost bot (https://github.com/yunohost-bot) I got this problem, and it has been canceled, anybody knows what's wrong by "bool"?

https://github.com/Shervin-QZ/dolibarr_ynh/blob/testing_yunosync/manifest.toml

```
============================================
 Package linter
 ============================================

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 291, in is_type
    [YunoHost App Package Linter]

 App packaging documentation - https://yunohost.org/packaging_apps
 App package example         - https://github.com/YunoHost/example_ynh
 Official helpers            - https://yunohost.org/packaging_apps_helpers

 If you believe this linter returns false negative (warnings / errors which shouldn't happen),
 please report them on https://github.com/YunoHost/package_linter/issues
    
  Analyzing app /tmp/package_check.sLkFHY/app_folder ...

    return self.TYPE_CHECKER.is_type(instance, type)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  _File "/usr/lib/python3/dist-packages/jsonschema/_types.py", line 120, in is_type
    raise UndefinedTypeCheck(type) from None
jsonschema.exceptions.UndefinedTypeCheck: Type 'bool' is unknown to this type checker_

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/www/yunorunner/package_check/./package_linter/package_linter.py", line 40, in <module>
    main()
  File "/var/www/yunorunner/package_check/./package_linter/package_linter.py", line 36, in main
    app.analyze()
  File "/var/www/yunorunner/package_check/package_linter/tests/test_app.py", line 259, in analyze
    self.manifest_.run_tests()
  File "/var/www/yunorunner/package_check/package_linter/lib/lib_package_linter.py", line 177, in run_tests
    this_test_reports = list(test(self))
                        ^^^^^^^^^^^^^^^^
  File "/var/www/yunorunner/package_check/package_linter/tests/test_manifest.py", line 524, in manifest_schema
    yield from validate_schema(
  File "/var/www/yunorunner/package_check/package_linter/lib/lib_package_linter.py", line 128, in validate_schema
    for error in v.iter_errors(data):
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 261, in iter_errors
    for error in errors:
  File "/usr/lib/python3/dist-packages/jsonschema/_validators.py", line 332, in properties
    yield from validator.descend(
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 278, in descend
    for error in self.evolve(schema=schema).iter_errors(instance):
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 261, in iter_errors
    for error in errors:
  File "/usr/lib/python3/dist-packages/jsonschema/_validators.py", line 25, in patternProperties
    yield from validator.descend(
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 278, in descend
    for error in self.evolve(schema=schema).iter_errors(instance):
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 261, in iter_errors
    for error in errors:
  File "/usr/lib/python3/dist-packages/jsonschema/_validators.py", line 332, in properties
    yield from validator.descend(
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 278, in descend
    for error in self.evolve(schema=schema).iter_errors(instance):
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 261, in iter_errors
    for error in errors:
  File "/usr/lib/python3/dist-packages/jsonschema/_validators.py", line 383, in oneOf
    errs = list(validator.descend(instance, subschema, schema_path=index))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 278, in descend
    for error in self.evolve(schema=schema).iter_errors(instance):
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 261, in iter_errors
    for error in errors:
  File "/usr/lib/python3/dist-packages/jsonschema/_validators.py", line 321, in type
    if not any(validator.is_type(instance, type) for type in types):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/jsonschema/_validators.py", line 321, in <genexpr>
    if not any(validator.is_type(instance, type) for type in types):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/jsonschema/validators.py", line 293, in is_type
    raise exceptions.UnknownType(type, instance, self.schema)
jsonschema.exceptions.UnknownType: Unknown type 'bool' for validator with schema:
    {'type': 'bool'}

**While checking instance:
    'syncyunohost_enabled'**
```

**Comment by Salamandar** [comment]
Indeed our json schema https://github.com/YunoHost/apps/blob/master/schemas/manifest.v2.schema.json might have a typo. Thanks for the info.

**Comment by Salamandar** [comment]
Fixed by https://github.com/YunoHost/apps/pull/2913
